.env.example

```
@@ -1,3 +1,4 @@
 STRIPE_SECRET=...
```

Also to prefix to give an indication. I think it's something like `sk_test` or `sk_live`

```suggestion
STRIPE_SECRET=sk_test...
```